.github/workflows/build-wheels-windows-dispatch_new.yml

```diff
@@ -0,0 +1,33 @@
+name: windows-dispatch
+
+on:
+  pull_request:
+  push:
+  workflow_dispatch:
+
+jobs:
+  build-python-wheels:
+    name: Build Python Wheels for windows-latest
+    runs-on: windows-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ['3.8.10']
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v3
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Get Python version
+        run: python --version
+
+      - name: Install build dependencis
+        run: pip install -r .\build_requirements.txt
+
+      - name: Build wheels for IDF
+        run: python .\Build-Wheels.py
```

Comment:
Optional: I would personally change the name of `Build-Wheels.py` file to something more standard in python like `build_wheels.py`. I can see that you wanted to keep the name from the original ps script but this looks a bit strange to me.

Reply:
That is true, thank you.